unitOneAssessmentPractice.js

```diff
@@ -70,8 +130,18 @@ let assert = require('assert')
 // Sample output:
 // {teacher: "Professor McGonagall", numberOfStudents: 20}
 
+const fewestStudents = arr => {
+  let smallest = arr[0]
+  arr.forEach(el=> {
+    if(el.age < smallest) {
```

Comment:
Here you are comparing the key of an object to a whole object (since you've set smallest to the first element of arr, which is an object). It doesn't work like that.
And we don't need the key "age" (technically we don't even have it), we need the key "numberOfStudents".